modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -120,22 +122,24 @@ If you don't see the *Connectors* tab, verify that you have a Titanium subscript
 | *Performance* a|
 
 * Average response time inbound
-* Average Response Time Grouped by Endpoint Inbound
+* Average response time grouped by endpoint - Inbound
 * Mule messages
 * Response time graphs: 99, 90, 75, and 50 percentile inbound
 * Average response time outbound
-* Average response time grouped by endpoint outbound
-* Response time graphs: 99, 90, 75, and 50 percentile outbound
+* Average response time grouped by endpoint - Outbound
+* Response time graphs: 99, 90, 75, and 50 percentile - Outbound
 //* Response Time graphs: 99, 90, 75, and 50 Percentile Inbound
 //* Average Response Time: Outbound, Grouped by Endpoint Outbound
```

Comment:
What about the capital letters and lack of hyphens in these: Percentile Inbound, Outbound, Grouped by Endpoint Outbound?

Reply:
These are commented out. I asked dev to let me know if they can be deleted. They're from before my time.